src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/utils.ts

```diff
@@ -101,5 +104,10 @@ export const getTxData = (tx: Transaction): TxData => {
     txData.recipient = tx.recipient
   }
 
+  if (sameString(tx.type, 'outgoing') && tx.symbol && sameString(tx.symbol, 'eth')) {
```

Comment:
Shouldn't we use `nativeCoin.symbol` here?

Reply:
Yes you are right, I fixed it thanks